package.json

```diff
@@ -17,7 +17,7 @@
     "test:types": "pnpm -r test:types",
     "test:knip": "knip",
     "test:versions": "installed-check -d --workspace-ignore='packages/fontless/examples/*'",
-    "postinstall": "simple-git-hooks install && pnpm --filter fontaine build"
+    "postinstall": "stale-dep -u && simple-git-hooks install && pnpm --filter fontaine build"
```

Comment:
this runs immediately after dependencies _are_ freshly installed, so I think this doesn't make sense here